api/src/main/java/org/openmrs/module/queue/api/QueueService.java

```diff
@@ -45,6 +49,7 @@ public interface QueueService {
 	 * @param queue the queue to be saved
 	 * @return saved {@link org.openmrs.module.queue.model.Queue}
 	 */
+	@Authorized({ PrivilegeConstants.ADD_QUEUES, PrivilegeConstants.EDIT_QUEUES })
```

Comment:
Does this require edit?

Reply:
yes, I think since it delegates `saveQueue()`, it should also requires the same privileges.